apps/studio/src/routes/editor/TopBar/OpenCode/index.tsx

```diff
@@ -97,24 +101,34 @@ const OpenCode = observer(() => {
     }
 
     return (
-        <div className="inline-flex items-center justify-center whitespace-nowrap rounded-md font-medium transition-colors focus-visible:outline-none h-8 border border-input shadow-sm bg-background hover:bg-background-onlook hover:text-accent-foreground text-xs space-x-0 p-0">
+        <div className="inline-flex items-center justify-center whitespace-nowrap overflow-hidden rounded-md font-medium transition-colors focus-visible:outline-none h-8 border border-input shadow-sm bg-background hover:bg-background-onlook hover:text-accent-foreground text-xs space-x-0 p-0">
             <Tooltip>
                 <TooltipTrigger asChild>
                     <div>
                         <DropdownMenu onOpenChange={(isOpen) => setIsDropdownOpen(isOpen)}>
                             <DropdownMenuTrigger
-                                className="flex flex-row items-center mr-2"
+                                className="flex flex-row items-center"
                                 asChild
                                 disabled={!instance && !root}
                             >
                                 <button
-                                    className="flex items-center text-smallPlus justify-center disabled:text-foreground-onlook h-full w-full min-w-[7.5rem] my-1 pl-2.5 hover:text-foreground-active/90"
```

Comment:
Keeping in mind that removing the min-w means that the height of the button changes on select, just making sure this is intended, otherwise this is a regression